resident/resident-service/src/main/java/io/mosip/resident/validator/ValidateSameData.java

```diff
@@ -18,11 +21,23 @@
 @Component
 public class ValidateSameData {
 
+    @Autowired
+    private Environment environment;
+
     private static final String AND_LANGUAGE_CODE = " and Language code: ";
+    public static final String EXCLUDED_ATTRIBUTE_LIST = "mosip.identity.get.excluded.attribute.list";
```

Comment:
please use string array to store list of values 
@Value("${listOfValues}")
private String[] valuesArray;

Reply:
mosip.identity.get.excluded.attribute.list=UIN,verifiedAttributes,IDSchemaVersion
https://github.com/mosip/mosip-config/pull/6858/files
@dhanendra06 raised PR for config also using like this should I change to string of array

Reply:
Changed code as per review comment.